next.config.js

```diff
@@ -1,5 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  output: 'export',
+  images: { unoptimized: true },
```

Comment:
this is required to make static export work. We just have to optimise our images ourselves.